vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
+  
+```{r, fig.align='center', fig.height=4, fig.width=6}
+bednetstimesteps <- c(1, 2) * year # The bednets will be distributed at the end of the first and the second year. 
 
 bednetparams <- set_bednets(
-  bednetparams,
-  timesteps = bednet_events$timestep,
-  coverages = c(.8, .8),
-  retention = 5 * year,
-  dn0 = matrix(c(.533, .45), nrow=2, ncol=1),
-  rn = matrix(c(.56, .5), nrow=2, ncol=1),
-  rnm = matrix(c(.24, .24), nrow=2, ncol=1),
-  gamman = rep(2.64 * 365, 2)
+  simparams,
+  timesteps = bednetstimesteps,
+  coverages = c(.8, .5),  # The first round is distributed to 80% of the population and the second round to 50%.
+  retention = 5 * year, # Nets are kept on average 5 years
+  dn0 = matrix(c(.533, .45), nrow=2, ncol=3), # Matrix of death probabilities for each mosquito species over time
+  rn = matrix(c(.56, .5), nrow=2, ncol=3), # Matrix of repelling probabilities for each mosquito species over time
+  rnm = matrix(c(.24, .24), nrow=2, ncol=3), # Matrix of minimum repelling probabilities for each mosquito species over time
+  gamman = rep(2.64 * 365, 2) # Vector of bednet half-lives for each distribution timestep
 )
 
 output <- run_simulation(sim_length, bednetparams)
-
-add_intervention_lines(plot_prevalence(output), bednet_events)
 ```
 
-## Indoor spraying
-
-We can do the same for IRS...
-
-```{r}
-sprayingparams <- simparams
-
-peak <- peak_season_offset(sprayingparams)
-spraying_events = data.frame(
-  timestep = c(1, 2) * year + peak - 3 * month,
-  name=c("Spraying 1", "Spraying 2")
-)
-
-sprayingparams <- set_spraying(
-  sprayingparams,
-  timesteps = spraying_events$timestep,
-  coverages = rep(.8, 2),
-  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=1),
-  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1),
-  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=1),
-  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=1),
-  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=1),
-  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1)
-)
+### Plot prevalence
+```{r, fig.align='center', fig.height=5, fig.width=7}
+plot_prev()
+```
+  
+## Visualisation 
+
+It is important to understand the difference between the input `coverages` argument of `malariasimulation::set_bednets()` and the resulting population bednet usage over time in the model. In the above example, bednets are distributed to a specified percentage of the population at each distribution round; however, the level of bednet usage may be different. 
```

Comment:
Last sentence re-arrange. 
When we set `coverages` in the above example to 0.8, we are telling the model to distribute bed nets to 80% of the population at year one, and to 50% of the population at year 2. However, the level of average bed net usage is not necessarily equal to 80% or 50%. Between these time points, bed net use will slowly decline over time (in reality this decline stems from things like disuse, holes in nets, lost nets, etc.).